Mvn build fails, clean checkout

[ERROR] Failed to execute goal on project openhab-addons-verify: Could not resolve dependencies for project org.openhab.addons:openhab-addons-verify:pom:1.8.0-SNAPSHOT: Failure to find org.openhab.core:openhab-core:xml:features:2.0.0-SNAPSHOT in https://jcenter.bintray.com/ was cached in the local repository, resolution will not be reattempted until the update interval of jcenter has elapsed or updates are forced -> [Help 1]

--- a/features/openhab-addons-verify/pom.xml
+++ b/features/openhab-addons-verify/pom.xml
@@ -90,36 +90,4 @@
             </plugin>
         </plugins>
     </build>
-
-    <repositories>
-        <repository>
-            <id>jcenter</id>
-            <name>JCenter Repository</name>
-            <url>https://jcenter.bintray.com/</url>
-        </repository>
-        <repository>
-            <id>eclipse</id>
-            <name>Eclipse Snapshot Repository</name>
-            <layout>default</layout>
-            <url>https://repo.eclipse.org/content/repositories/snapshots/</url>
-            <snapshots>
-                <enabled>true</enabled>
-            </snapshots>
-        </repository>
-        <repository>
-           <id>tmp-jaxrs-swagger</id>
-           <url>https://github.com/maggu2810/openhab2-mvn-repo/raw/jaxrs-swagger</url>
-           <snapshots>
-              <enabled>true</enabled>
-            </snapshots>
-        </repository>
-        <repository>
-           <id>tmp-shk</id>
-           <url>https://github.com/maggu2810/openhab2-mvn-repo/raw/shk</url>
-           <snapshots>
-              <enabled>true</enabled>
-            </snapshots>
-        </repository>
-    </repositories>
-
 </project>

--- a/features/pom.xml
+++ b/features/pom.xml
@@ -70,6 +70,14 @@
                 <enabled>true</enabled>
             </snapshots>
         </repository>
+        <repository>
+            <id>jfrog</id>
+            <name>JFrog OSS Repository</name>
+            <url>http://oss.jfrog.org/libs-snapshot/</url>
+            <snapshots>
+                <enabled>true</enabled>
+            </snapshots>
+        </repository>
     </repositories>
 
 

